ui/src/features/Editor/components/OverlayUI/components/DebugPanel/index.tsx

```diff
@@ -0,0 +1,218 @@
+              <TabsTrigger
+                className="gap-1 bg-card font-thin"
+                value="debug-logs"
+                onClick={handleTabChange}>
+                <CodeIcon />
+                <p className="text-sm font-thin select-none">
+                  {t("Workflow Logs")}
+                </p>
+              </TabsTrigger>
+              <TabsTrigger
+                className="gap-1 bg-card font-thin"
+                value="debug-viewer"
+                disabled={!dataURLs?.length}
+                onClick={handleTabChange}>
+                <EyeIcon />
+                <p className="text-sm font-thin select-none">
+                  {t("Data Preview")}
+                </p>
+              </TabsTrigger>
+            </TabsList>
+          </div>
+        </div>
+        <div className="mr-[120px] flex flex-1 items-center justify-center gap-2">
+          <TerminalIcon />
+          <p className="text-sm font-thin select-none">{t("Debug Run")}</p>
+        </div>
+        <div className="flex items-center gap-2 p-1">
+          {!fullscreenDebug && (
+            <div
+              className="cursor-pointer rounded p-1 hover:bg-primary"
+              onClick={handleMinimize}>
+              {minimized ? (
+                <CaretUpIcon weight="light" />
+              ) : (
+                <MinusIcon weight="light" />
+              )}
+            </div>
+          )}
+          {!minimized && !fullscreenDebug && (
+            <div
+              className="cursor-pointer rounded p-1 hover:bg-primary"
+              onClick={handleExpand}>
+              {expanded ? (
+                <CaretDownIcon weight="light" />
+              ) : (
+                <CaretUpIcon weight="light" />
+              )}
+            </div>
+          )}
+          {!minimized && (
+            <div
+              className="cursor-pointer rounded p-1 hover:bg-primary"
+              onClick={onFullScreenExpand}>
+              {fullscreenDebug ? (
+                <CornersInIcon weight="light" />
+              ) : (
+                <CornersOutIcon weight="light" />
+              )}
+            </div>
+          )}
+        </div>
+      </div>
+      <TabsContent
+        value="debug-logs"
+        className="h-[calc(100%-35px)] overflow-scroll"
+        forceMount={debugJobIdRef.current !== debugJobId ? undefined : true}
+        hidden={tabValue !== "debug-logs"}>
+        <DebugLogs debugJobId={debugJobId} />{" "}
+      </TabsContent>
+      {dataURLs && (
+        <TabsContent
+          value="debug-viewer"
+          forceMount={true}
+          hidden={tabValue !== "debug-viewer"}
+          className="h-[calc(100%-35px)] overflow-scroll">
+          <ResizablePanelGroup className="h-full w-full" direction="horizontal">
+            <ResizablePanel defaultSize={70} minSize={20}>
+              <Tabs defaultValue="data-viewer">
+                <div className="relative flex w-fit items-center p-1">
```

Comment:
Again relative and flex wrapping a single div (with no ascertainable absolute element